forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -182,6 +195,26 @@ interface DeviceAuthenticator {
         return Calendar.getInstance().time
     }
 
+    /** Validate custom claims
+    * @param  customClaims: A map of custom claims to be validated
+    * @return Boolean value indicating whether the custom claims are valid or not
+     */
+    fun validateCustomClaims(customClaims: Map<String, Any>): Boolean {
+        val registeredKeys = listOf(
```

Comment:
May use static for the registeredKeys.

Reply:
what's static version in kotlin? seems like it's a java thing no?

Reply:
companion object

Reply:
changed